otherlibs/win32unix/createprocess.c

```diff
@@ -24,33 +24,45 @@
 
 static int win_has_console(void);
 
+static int ensure_inheritable(HANDLE h, HANDLE * hStd, HANDLE * to_close)
```

Comment:
Can you add a brief comment above the function definition explaining what the function does and what each argument means?

Reply:
Done, hope it's clearer.